app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -211,6 +250,158 @@ def get_locations
           location.match(VALID_LOCATION).to_s
         end
       end
+
+      def parse_stack(deployment)
+        name = deployment.fetch_path('name')
+        uid = resource_uid(@subscription_id,
+                           deployment.fetch_path('resourceGroup'),
+                           TYPE_DEPLOYMENT,
+                           name)
+        child_stacks, resources = find_stack_resources(deployment)
+        new_result = {
+          :type        => ManageIQ::Providers::Azure::CloudManager::OrchestrationStack.name,
+          :ems_ref     => deployment.fetch_path('id'),
+          :name        => name,
+          :description => name,
+          :status      => deployment.fetch_path('properties', 'provisioningState'),
+          :children    => child_stacks,
+          :resources   => resources,
+          :outputs     => find_stack_outputs(deployment),
+          :parameters  => find_stack_parameters(deployment),
+
+          :orchestration_template => find_stack_template(deployment)
```

Comment:
blank line is to break the alignment, otherwise all above lines have to shift to right. I learned this from examples in other parser code. robcop may complain this blank line, but I don't know what's the best.